test/Release/GetReleaseToggles/ResponseToggleTest.php

```diff
@@ -4,12 +4,30 @@
 class ResponseToggleTest extends \PHPUnit_Framework_TestCase
 {
     const NAME = 'name';
+    const ID = 'id';
+    const DESC = 'description';
 
     /**
      * @test
      */
     public function GivenToggleName_WhenGetName_ThenReturnsName()
     {
-        $this->assertEquals( self::NAME, ( new ResponseToggle( self::NAME ) )->getName() );
+        $this->assertEquals( self::NAME, ( new ResponseToggle( null, self::NAME, null ) )->getName() );
+    }
+
+    /**
+     * @test
+     */
+    public function givenToggleId_WhenGetId_IdReturnedIsCorrect()
+    {
+        $this->assertEquals( self::ID, ( new ResponseToggle( self::ID, null, null ) )->getId() );
+    }
+
+    /**
+     * @test
+     */
+    public function given__()
```

Comment:
Was this intentional, or did you forget to give the "When" and "Return" for this case?